src/main/java/core/basesyntax/dao/FruitTransactionDaoImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FruitTransactionDaoImpl implements FruitTransactionDao {
+
+    @Override
+    public List<String[]> getAllTransactions(String fileName) {
+        try {
+            List<String> stringList = Files.readAllLines(Path.of(fileName));
+            stringList.remove(0);
+            return stringList.stream()
+                    .map(f -> f.split(","))
```

Comment:
use skip(1) in stream for this
extract 1 to OFFSET constant to make it move visible
f  - bad naming, avoid short one-letter names even in stream chains
"," - better extract literal to constant, like SEPARATOR